### DISCUSSION TITLE
it keeps tellin me I dont have the US 1.2 release EXE

### DISCUSSION THREAD

**Discussion by ComradeStonedSoviet** [DADi590/F1DP#4] [Q&A]
I have Team X update, and it still keeps tellin me that Im missing the perameter regarding the 1.2 US version Dos Release, its stressing me out, pls help
![image](https://user-images.githubusercontent.com/113244566/189513737-6bf5ecf8-c1dd-4f54-b9cf-5fb14e7d0d0e.png)

**Comment by Allanon41** [comment]
I can confirm that this works on DOS ver. US 1.2 from Team X.

**Comment by Edw590** [comment]
Oh damn. I never saw this! I'm sorry... Hopefully it's working now. It should always work as long as you have the original EXE (not a modified one, must be the original one).

EDIT: ah, unless it's this problem: https://github.com/Edw590/F1DP/issues/5. Then I hope it worked in the end. You weren't giving it the file in one of the 2 supposed ways (that's what the error explains - no file was given. Not that it's the wrong one. Instead none was given).

**Reply by Allanon41** [reply]
Actually the 1.2 patch from Team X has the 1.2 .exe but the Tandi script from Team X causes a crash. Don't know about others.![image](https://github.com/Edw590/F1DP/assets/32942104/920ba8fc-41ec-4231-abb5-31fdeade20d4)

**Reply by Edw590** [reply]
Right... Interesting. Can you test on Windows and see if it also crashes the game there? (Maybe could be a specific case of yours or something that makes the game crash?) Just run the Windows EXE on that version, instead of running the DOS one (need to be in Windows though).

In any case, would you mind trying patch 1.3 on it? I checked the tandi.int files and they're the same in patch 1.2 and 1.2.1, but it's different in 1.3. So maybe try to install patch 1.3 to see if the error goes away? They say to install it only on top of patch 1.2, so make sure you didn't install 1.2.1 too.

Btw, how do you know it's that file? Removed files one by one until the error disappeared? Or through some other method?

EDIT: the file is different in 1.3 because they fixed some things there. Could be worth trying. Remaining in 1.2 when 1.3 or 1.2.1 are available may not be the best idea (if you were/are doing that).

**Reply by Allanon41** [reply]
Talking to Tandi was the first time I encountered an error. I dont use Windows as I'm basically mobile (ios)only. Playing this with RetroArch using Dosbox-Pure. I did a whole new install using these mods done in order:
![image](https://github.com/Edw590/F1DP/assets/32942104/bdd2f6f3-50b3-43fb-87a4-cf7f33f1421e)

Been playing fine so far, just completed Vault-15. Be nice to use your modifications (from sfall) like  double-click to run and dont turn off sneak as well as the rest. I'll try again with my setup using your 1.2 modification. Theres also this mod from Russian modders that is a compilation of all fallout 1 mods I want to try:https://www.nuclear-city.com/topic/992-fallout-1-unofficial-patch-136/

**Reply by Edw590** [reply]
Ah ok, no problem! Tell me how it got after you test that then! If the problem gets fixed by itself just by updating the patches, cool. If not... I would probably need your setup or at least the save to have the crash with me too - except my time is (very?) limited because of university, so it might take some time for me to analyze the issue (and hopefully be able to fix it).

**Reply by Allanon41** [reply]
I tracked down where I got the crash from. It's related to the sFall option of having numbered (1,2,3,etc) dialog options. Specifically: NumbersinDialogue=

There's also a related crash with preventing from using 0 in dialogue:
DialogueFix=

I dont know how to upload files for save, but if you run to Tandi with NumbersinDialogue= enabled you should get the crash unless its specifically a RetroArch Dosbox-Pure thing.
